.github/stale.yml

```diff
@@ -0,0 +1,28 @@
+# Configuration for probot-stale - https://github.com/probot/stale
```

Comment:
@apb7 I just noticed I haven't seen any messages from probot. Is it actually working?

Reply:
@seanlip: We have not installed it yet for the oppia repository. I will be doing this in Milestone 2. Thanks!

Reply:
OK, thanks!